gallery/src/components/NxDropdownIconOnly/NxDropdownIconOnlyPage.tsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxTextLink, NxCode, NxH3, NxP, NxTile, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+
+import NxDropdownIconOnlyNavigationExample from './NxDropdownIconOnlyNavigationExample';
+import NxDropdownIconOnlyLinksExample from './NxDropdownIconOnlyLinksExample';
+import NxDropdownIconOnlyDisabledExample from './NxDropdownIconOnlyDisabledExample';
+
+const nxDropdownIconOnlyNavigationExampleCode = require('./NxDropdownIconOnlyNavigationExample?raw'),
+    nxDropdownIconOnlyLinksExampleCode = require('./NxDropdownIconOnlyLinksExample?raw'),
+    nxDropdownIconOnlyDisabledExampleCode = require('./NxDropdownIconOnlyLinksExample?raw');
+
+const NxDropdownPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A variation of the dropdown menu component which uses a single icon as its toggle.
+        <NxCode>NxDropdownIconOnly</NxCode> shares all of the same props as{' '}
+        <NxTextLink href="#/pages/NxDropdown" className="nx-dropdown-button">
+          NxDropdown
+        </NxTextLink>{' '}
+        as well as the ones listed below. Like <NxCode>NxDropdown</NxCode> this is for menus of nagivation links or
+        action-triggering buttons.  It is <em>not</em> a form select field. By default the dropdown can display a
+        maximum of 10 items before it scrolls the contents of the dropdown menu.
+      </NxP>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Props:</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxTable>
+          <NxTable.Head>
+            <NxTable.Row>
+              <NxTable.Cell>Prop</NxTable.Cell>
+              <NxTable.Cell>Type</NxTable.Cell>
+              <NxTable.Cell>Required</NxTable.Cell>
+              <NxTable.Cell>Details</NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Head>
+          <NxTable.Body>
+            <NxTable.Row>
+              <NxTable.Cell>icon</NxTable.Cell>
+              <NxTable.Cell>string | VDOM</NxTable.Cell>
```

Comment:
This works as a string?

Reply:
> This works as a string?

https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/94abc6ae861cbd13147bbbf25d88876f640c708b